src/app/tests/suites/certification/Test_TC_APPOBSERVER_1_13.yaml

```diff
@@ -22,73 +22,73 @@ PICS:
-          [1702624923.224578][5892:5894] CHIP:EM: Flushed pending ack for MessageCounter:113660697 on exchange 5445i
-      disabled: true
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
 
     - label: "Step 2: TH reads the FeatureMap attribute from the DUT"
-      verification: |
-          ./chip-tool contentappobserver read feature-map 1 1
-          On TH, verify that DUT sends the FeatureMap attribute has the value 0 or returns a general error that the attribute is not supported
-          [1702624956.728609][5896:5898] CHIP:DMG:         SuppressResponse = true,
-          [1702624956.728667][5896:5898] CHIP:DMG:         InteractionModelRevision = 11
-          [1702624956.728720][5896:5898] CHIP:DMG: }
-          [1702624956.729021][5896:5898] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0510 Attribute 0x0000_FFFC DataVersion: 2272699478
-          [1702624956.729138][5896:5898] CHIP:TOO:   FeatureMap: 0
-          [1702624956.729564][5896:5898] CHIP:EM: <<< [E:26717i S:26806 M:11352307 (Ack:31492418)] (S) Msg TX to 1:0000000000000001 [EDAD] [UDP:[fe80::e65f:1ff:fe49:ae1a%wlan0]:5640] --- Type 0000:10 (SecureChannel:StandaloneAck)
-      disabled: true
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "Step 3: TH reads the AttributeList attribute from the DUT"
+      PICS: PICS_EVENT_LIST_ENABLED
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [65528, 65529, 65530, 65531, 65532, 65533]
 
     - label: "Step 3: TH reads the AttributeList attribute from the DUT"
-      verification: |
-          ./chip-tool contentappobserver read  attribute-list 1 1
-          Verify " AttributeList " value consists the list of mandatory attributes (65533, 65532, 65531, 65529, 65528) on the TH(Chip-tool) Log:
-          [1702624983.367693][5900:5902] CHIP:DMG: }
-          [1702624983.368044][5900:5902] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0510 Attribute 0x0000_FFFB DataVersion: 2272699478
-          [1702624983.368485][5900:5902] CHIP:TOO:   AttributeList: 5 entries
-          [1702624983.368564][5900:5902] CHIP:TOO:     [1]: 65528
-          [1702624983.368624][5900:5902] CHIP:TOO:     [2]: 65529
-          [1702624983.368683][5900:5902] CHIP:TOO:     [3]: 65531
-          [1702624983.368741][5900:5902] CHIP:TOO:     [4]: 65532
-          [1702624983.368798][5900:5902] CHIP:TOO:     [5]: 65533
-          [1702624983.369236][5900:5902] CHIP:EM: <<< [E:55491i S:35892 M:103959395 (Ack:145764997)] (S) Msg TX to 1:0000000000000001 [EDAD] [UDP:[fe80::e65f:1ff:fe49:ae1a%wlan0]:5640] --- Type 0000:10 (SecureChannel:StandaloneAck)
-      disabled: true
+      PICS: "!PICS_EVENT_LIST_ENABLED"
```

Comment:
Contains only checks that the values are part of the list, not that values not in the list are excluded. So these two pics conditions can effectively be reduced to a single condition